### PR TITLE
Day6

### PR DESCRIPTION
**Pull request by ShahnamehHon**
Day 6 hw

### CODE REVIEW

File: src/main/java/Zoo.java [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-        myZoo.setName("Ailuropoda melanoleuca");
-        String printName = myZoo.getName();
-        myZoo.setAge(8);
-        int printAge = myZoo.getAge();
-        myZoo.setWeight(285.05);
-        double printWeight = myZoo.getWeight();
-        System.out.println("Scientific name: " + printName + "\n" + "Age: " + printAge  + "\n" + "Weight: " + printWeight);
-        //Homework
-    }
-}
```
Comment (ydanylchenko) [inline src/main/java/Zoo.java:13]: @ShahnamehHon You deleted the Zoo.java file in this branch. On merge of this Pull request, the Zoo.java will be removed from origin/master as well. Is it done intentionally?

File: src/main/java/Panda.java [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
-
-    public double getWeight() {
-        return weight;
-    }
-
-    public void setWeight(double weight) {
-        this.weight = weight;
-    }
-}
-
```
Comment (ydanylchenko) [inline src/main/java/Panda.java:32]: @ShahnamehHon Same as with Zoo.java. You deleted it. Is it done intentionally?

File: src/main/java/Orchid.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+public class Orchid {
+    String name;
+
+    int cutOrchid;
+
+    double species;
+
+    Orchid(){
```
Comment (ydanylchenko) [inline src/main/java/Orchid.java:8]: First constructor

File: src/main/java/Orchid.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    }
+
+    public Orchid(String name, int cutOrchid, double species) {
```
Comment (ydanylchenko) [inline src/main/java/Orchid.java:19]: Second constructor

**Review by ydanylchenko — COMMENTED** [review]
@ShahnamehHon  Good job, but it's not fully completed. Could you please add 6 more missing constructors in addition to the two that you have added

File: src/main/java/Zoo.java (hunk 1 above)
Comment (ShahnamehHon) [reply src/main/java/Zoo.java:13]: yes it was. i created two add files in day6 branch

File: src/main/java/Panda.java (hunk 2 above)
Comment (ShahnamehHon) [reply src/main/java/Panda.java:32]: Same with this one another file created in branch day6

File: src/main/java/Orchid.java (hunk 3 above)
Comment (ShahnamehHon) [reply src/main/java/Orchid.java:8]: ok

File: src/main/java/Orchid.java (hunk 4 above)
Comment (ShahnamehHon) [reply src/main/java/Orchid.java:19]: ok

Comment (ShahnamehHon) [reply src/main/java/Orchid.java:19]: please check second file in day6 branch

File: src/main/java/Zoo.java (hunk 1 above)
Comment (ydanylchenko) [reply src/main/java/Zoo.java:13]: ok, cool

File: src/main/java/Panda.java (hunk 2 above)
Comment (ydanylchenko) [reply src/main/java/Panda.java:32]: ok, cool

File: src/main/java/Orchid.java (hunk 4 above)
Comment (ydanylchenko) [reply src/main/java/Orchid.java:19]: In the second file you are using the two constructors that you created, the Orchid() and the Orchid(String name, int cutOrchid, double species)
But as per the homework you should have 8 constructors

Comment (ydanylchenko) [reply src/main/java/Orchid.java:19]: E.g. you need to be able to create object of class with just name provided, i.e. the public Orchid(String name){} constructor should be added, and in similar way for all the other combinations of name, cut and species

**Pull request merged**